docs-src/concepts/what-is-the-temporal-web-ui.md

```diff
@@ -125,9 +121,9 @@ Displays the following information:
 
 ### Import Event History
 
-Navigate to `/import` to import an Event History in JSON format and populate a Workflow details page.
+Allows the Event History to be imported in JSON format for populating a Workflow details page.
```

Comment:
I'm not sure whether this feature is available via Web UI anymore.

Reply:
Went and checked, and sure enough, it is not! I'll go change this.